config-demo.toml

```diff
@@ -10,3 +10,6 @@
     password = "listmonk"
     database = "listmonk"
     ssl_mode = "disable"
+    max_open = 25
+    max_idle = 25
+    max_conn_lifetime = 300 # 5min or 300s.
```

Comment:
A timestring here is better. eg: `max_conn_lifetime = "300s"`

Reply:
Yep. Just realised it was setting it as `300ns` otherwise. Phew. :sweat_smile: 

Fixed.